badge-maker/lib/xml.js

```diff
@@ -73,4 +73,27 @@ class XmlElement {
   }
 }
 
-module.exports = { escapeXml, stripXmlWhitespace, XmlElement }
+class NullElement {
+  // TODO: we can remove this later
+  render() {
+    return ''
+  }
+}
```

Comment:
This was helpful as an intermediate step in the refactoring but it gets deleted in a later commit.